code/game/turfs/open/_open.dm

```diff
@@ -14,6 +14,13 @@
 	var/clawfootstep = null
 	var/heavyfootstep = null
 
+/turf/open/CanPass(atom/movable/A, turf/T)
+	if(isliving(A))
+		var/turf/AT = get_turf(A)
+		if(AT && AT.turf_height - turf_height <= -TURF_HEIGHT_BLOCK_THRESHOLD)
```

Comment:
As previously mentioned, this proc is already overriden in pool.dm. You need to merge them or remove the one in pool.dm. same with `/turf/open/MouseDrop_T(mob/living/M, mob/living/user)`

Reply:
Are you sure you meant ``/turf/open/MouseDrop_T(mob/living/M, mob/living/user)``?